ccsds/TM-Packet.acn

```diff
@@ -40,7 +40,16 @@ TM-PacketDataField <INTEGER:secondaryHeaderFlag> []
 
 TM-DataWithSecondaryHeader []
 {
-    secondaryHeader [],
+    secondaryHeader [] {
+        tmPacketPusVersionNumber [],
+        spacecraftTimeReferenceStatus [],
+        serviceType-ID MessageType.ServiceType-ID [],
+        messageSubtype-ID MessageType.MessageSubtype-ID [],
+        messageTimeCounter [size 16, encoding pos-int, endianness big],
```

Comment:
message-type-counter?

Reply:
hmm, true, looks like typo in the original version, on it